react/features/toolbox/components/web/Toolbox.js

```diff
@@ -996,6 +1004,10 @@ class Toolbox extends Component<Props, State> {
                 key = 'settings'
                 showLabel = { true }
                 visible = { this._shouldShowButton('settings') } />,
+            <MuteEveryoneButton
+                key = 'mute-everyone'
+                showLabel = { true }
+                visible = { _isModerator } />,
```

Comment:
Why this button is not controlled by the interface_config?

Reply:
updated